low-level/framework/base/module_thread.py

```diff
@@ -25,9 +25,6 @@
 import threading
 import time
 from sched import scheduler
-
-from framework.utils.service_logging import logger
```

Comment:
why removed ? logger getting used in code down below

Reply:
Saw that logger is imported twice in the file. so deleted one.